MDANSE/Src/MDANSE/MolecularDynamics/Trajectory.py

```diff
@@ -54,11 +60,223 @@ def __init__(self, filename, trajectory_format=None):
         if self._format is None:
             self.guess_correct_format()
         if self._format is None:
-            self._format = 'MDANSE'
-        self._trajectory = self.open_trajectory(self._format)
+            self._format = "MDANSE"
+        if self._format not in ("mock",):
+            self._trajectory = self.open_trajectory(self._format)
         self._min_span = np.zeros(3)
         self._max_span = np.zeros(3)
+        self._grouping_level = "atom"
+        self._group_lookup = {}
+        self._element_from_label = {}
+        self._atom_names = {}
         self._atom_cache = {}
+        self._selection = []
+        self.selection_getter = None
+        self._transmutation = {}
+        self._transmuted_types = []
+
+    @property
+    def atom_indices(self) -> Sequence[int]:
```

Comment:
```suggestion
    def atom_indices(self) -> list[int]:
```
Since this is explicitly a list.